<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestorUsuarios</class>
 <widget class="QWidget" name="GestorUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestor de Usuarios</string>
  </property>
  <property name="styleSheet">
   <string>QWidget#GestorUsuarios {
    background-color: #f5f5f5;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxRegistro">
     <property name="title">
      <string/>
     </property>
     <property name="styleSheet">
      <string>QGroupBox {
    border: 2px solid #e0e0e0;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 10px;
    background-color: #ffffff;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_registro">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTituloRegistro">
        <property name="text">
         <string>Registrar Nuevo Usuario</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_formulario">
        <property name="horizontalSpacing">
         <number>15</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelNombre">
          <property name="text">
           <string>Nombre:</string>
          </property>
          <property name="styleSheet">
           <string>QLabel { font-weight: bold; }</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditNombre">
          <property name="styleSheet">
           <string>QLineEdit {
    padding: 8px;
    border: 2px solid #e0e0e0;
    border-radius: 5px;
    font-size: 12px;
    background-color: white;
    color: black;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelEmail">
          <property name="text">
           <string>Email:</string>
          </property>
          <property name="styleSheet">
           <string>QLabel { font-weight: bold; }</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEditEmail">
          <property name="styleSheet">
           <string>QLineEdit {
    padding: 8px;
    border: 2px solid #e0e0e0;
    border-radius: 5px;
    font-size: 12px;
    background-color: white;
    color: black;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelTelefono">
          <property name="text">
           <string>Teléfono:</string>
          </property>
          <property name="styleSheet">
           <string>QLabel { font-weight: bold; }</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditTelefono">
          <property name="styleSheet">
           <string>QLineEdit {
    padding: 8px;
    border: 2px solid #e0e0e0;
    border-radius: 5px;
    font-size: 12px;
    background-color: white;
    color: black;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="labelDireccion">
          <property name="text">
           <string>Dirección:</string>
          </property>
          <property name="styleSheet">
           <string>QLabel { font-weight: bold; }</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEditDireccion">
          <property name="styleSheet">
           <string>QLineEdit {
    padding: 8px;
    border: 2px solid #e0e0e0;
    border-radius: 5px;
    font-size: 12px;
    background-color: white;
    color: black;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_botones">
        <item>
         <spacer name="horizontalSpacer_left">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRegistrar">
          <property name="text">
           <string>Registrar Usuario</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLimpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_right">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLista">
     <property name="title">
      <string/>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>280</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>QGroupBox {
    border: 2px solid #e0e0e0;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 10px;
    background-color: #ffffff;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_lista">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_titulo">
        <item>
         <widget class="QLabel" name="labelTituloLista">
          <property name="text">
           <string>Lista de Usuarios Registrados</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_titulo">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonActualizar">
          <property name="text">
           <string>Actualizar Lista</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetUsuarios">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
